boilerplate/src/app/_layout.tsx

```diff
@@ -15,11 +15,9 @@ if (__DEV__) {
   // Load Reactotron configuration in development. We don't want to
   // include this in our production bundle, so we are using `if (__DEV__)`
   // to only execute this in development.
-  require("src/devtools/ReactotronConfig.ts")
+  require("../devtools/ReactotronConfig")
```

Comment:
Ah this is a missed update to the template when we changed the path alias in tsconfig. Can we use that instead? `@/devtools...` should work I think

Reply:
Good catch! [fix(boilerplate): update ReactotronConfig import path to use alias](https://github.com/infinitered/ignite/pull/2999/commits/8e4f5b998ac4054e1883b1f5a46709f0371c8bc1)